src/main/java/kohn/maze/gui/Maze.java

```diff
@@ -0,0 +1,261 @@
+package kohn.maze.gui;
+
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.Random;
+import java.util.Stack;
+
+public class Maze extends JComponent {
```

Comment:
You should be creating a MazeView that has a Maze

Reply:
Oh I did that, but it doesn't display, so I committed a version that did. II just pushed the one with mazeView.

Reply:
Nevermind, it works now.